ucs-exporter.py

```diff
@@ -46,13 +52,16 @@ def start_collector(params):
     """
     creds = {"username": params['user'], "master_password": params['master_password']}
     inventory_file = params["inventory"]
-    ucsm_collector = UcsmCollector(creds=creds, inventory_file=inventory_file)
-    REGISTRY.register(ucsm_collector)
+
+    # Register collectors
+    for collector, class_name in COLLECTORS.items():
+        REGISTRY.register(getattr(import_module("collectors.{}".format(
+                                collector)), class_name)(creds, inventory_file))
```

Comment:
You don't need to use a dict here. Your dict looks like:

```
{ "a": "a", "b" : "b"}
```

and this should be a list [ "a", "b"]. Then loop through it.

Reply:
This is done.